views/update/update-check-your-answers.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Check your answers before completing your update - {{ SERVICE_NAME }} - GOV.UK
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+
+      <h1 class="govuk-heading-xl">Check your answers before sending your application (NOT LIVE)</h1>
```

Comment:
Do we want to see "NOT LIVE" when the update flag is enabled?

Reply:
Same as the other comment, I vaguely remember either a content designer or someone from product saying if we can mark all our pages as NOT LIVE in some way so this is what we've been doing with all of them so far. Then we'll update before all this goes live.